app/mailers/user_mailer.rb

```diff
@@ -5,10 +5,10 @@ class UserMailer < ApplicationMailer
   #
   #   en.user_mailer.account_activation.subject
   #
-  # def account_activation(user)
```

Comment:
delete this line. not comment

Reply:
I already remove the comment and need to use this code to activate the account for the user